src/main/java/org/server/mbtiliarserver/question/domain/Question.java

```diff
@@ -0,0 +1,20 @@
+package org.server.mbtiliarserver.question.domain;
+
+public class Question {
+
+    Long id;
+    String question;
```

Comment:
접근 제어자가 설정이 되지 않았네요 ㅜ 
```suggestion
    private Long id;
    private String question;
```